test/explicit-resource-management/main.test.ts

```diff
@@ -1,52 +1,57 @@
-
+import { expect } from 'chai';
```

Comment:
This file was not linted until now because eslint's TS parser only supported TS 5.2 until typescript-eslint@8.x.